README.md

```diff
@@ -1,39 +1,45 @@
 # fablicop
 
 fablicop is a RuboCop configration gem.
+It assumes your project is using Ruby on Rails, so some Cops prefixed with `Rails` are enabled out of the box.
+In other words, it's not appropriate to use it with non-Rails projects.
 
-## Usage
+## Installation
```

Comment:
I moved the `Installation` section here because I think many are interested in how to install first.